packages/frontend/web/src/components/StarRating.tsx

```diff
@@ -0,0 +1,12 @@
+import type { StarRatingProps } from '@app/types';
+
+import Star from './Star';
+
+export default function StarRating({ starCount }: StarRatingProps) {
+  const stars = [];
+  for (let index = 0; index < starCount; index++) {
+    stars.push(<Star key={index} index={index} />);
+  }
+
+  return <div className='mt-2 flex justify-center'>{stars}</div>;
```

Comment:
Pourquoi ne pas faire un `Array.from` directement dans le return à la place d'une boucle for ?